pkg/util/docker/docker_test.go

```diff
@@ -120,31 +120,33 @@ func TestResolveImageNameFromContainer(t *testing.T) {
 		expectedImage string
 	}{
 		{
-			name: "test empty Image",
+			name: "test empty config image name",
 			input: types.ContainerJSON{
-				ContainerJSONBase: &types.ContainerJSONBase{Image: ""},
```

Comment:
could this case ever happen?

Reply:
not sure, we can probably drop this test case, the case of when `.Image` is ignored is covered by the next test case.